app/models/application.rb

```diff
@@ -149,4 +136,12 @@ def sponsor_pick?
       value.to_s != '0' ? flags.concat([flag.to_s]).uniq : flags.delete(flag.to_s)
     end
   end
+
+  def project1
+    Project.find_by(id: application_data['project1_id'])
```

Comment:
How do you imagine this? _"Just"_ as a data transfer object or as a model persisted to the database?

But related to that: it's super hard to render these types of application overview views. I'd need to preload the teams and projects associated with an application in order to make this more sane (snappy would be the wrong word). However, I only know how to do this in raw sql...